f5_openstack_agent/lbaasv2/drivers/bigip/resource_manager.py

```diff
@@ -1154,11 +1155,20 @@ def _create_single(self, resource, service, **kwargs):
                 name=urllib.quote(pool_payload['name']),
                 partition=pool_payload['partition']
             )
-            pool_resource.members_s.members.create(**payload)
-            self._shrink_payload(pool_payload,
-                                 keys_to_keep=['partition',
-                                               'name', 'loadBalancingMode'])
-            self._pool_mgr._update(bigip, pool_payload, None, None, service)
+
+            try:
+                pool_resource.members_s.members.create(**payload)
```

Comment:
Do we only need to try/catch this line?

Reply:
Yes, it is not necessary to continue, if we get HTTP 409.